script/Arcueid/Day 9/09_00_ARC09_4_2.txt

```diff
@@ -971,6 +971,7 @@ I give her a wave and set off home.
 [sha:6b5eec75a1a0b63e27e08afe1e236fcc0d330c93]{
 -- Page 299, Offset 22124.
 -- と、アルクェイドに呼び止められた。
+//instead of 'calls me to a halt' consider 'brings me to a halt' or 'stops me in my tracks'
```

Comment:
Hmm, she is doing it at a distance by calling out to him, so I think it might be ok? The other two feel a bit more physically arresting, at least to me.